solution/backend/common/serializers.py

```diff
@@ -0,0 +1,19 @@
+from drf_spectacular.utils import extend_schema_field
+from rest_framework import serializers
+
+from resources.serializers.categories import AbstractCategoryPolymorphicSerializer, MetaCategorySerializer
+from resources.serializers.locations import AbstractLocationPolymorphicSerializer, MetaLocationSerializer
+
+
+class DetailsSerializer(serializers.Serializer):
```

Comment:
if we include subjects  into subject details we can kind of use this as a central "include or exclude" details.   Locations is currently shared amongst different types but subjects could also be included here.